=== command
SEMGREP_USER_AGENT_APPEND="pytest" SEMGREP_SETTINGS_FILE="<MASKED>" SEMGREP_VERSION_CACHE_PATH="<MASKED>" SEMGREP_ENABLE_VERSION_CHECK="0" SEMGREP_SEND_METRICS="off" semgrep --test --strict --config rules/fixtest/test4.yaml targets/fixtest/test4.py
=== end of command

=== exit code
1
=== end of exit code

=== stdout - plain
No unit tests found. See https://semgrep.dev/docs/writing-rules/testing-rules
0/1: 1 fix tests did not pass: 
--------------------------------------------------------------------------------
	âœ– targets/fixtest/test4.fixed.py <> autofix applied to targets/fixtest/test4.py 

	---
	+++
	@@ -2 +2 @@
	-sys.exit(1)
	+exit(1)
	@@ -5 +5 @@
	-sys.exit(2)
	+exit(2)



--------------------------------------------------------------------------------
The following config files produced errors:
	rules/fixtest/test4.yaml: STATIC ERROR: rules/fixtest/test4.yaml:1:6: did not expect: ":"


=== end of stdout - plain

=== stderr - plain

=== end of stderr - plain